app/javascript/components/Profile.js

```diff
@@ -0,0 +1,209 @@
+                  type="text"
+                  ref={node => {
+                    input = node;
+                  }}
+                  placeholder="Personal Access Token"
+                  aria-label="Access Token"
+                />
+                <button
+                  className="flex-no-shrink bg-teal hover:bg-teal-dark border-teal hover:border-teal-dark text-sm border-4 text-white py-1 px-2 rounded"
+                  type="submit"
+                >
+                  Update Token
+                </button>
+              </div>
+            </form>
+          </div>
+        );
+      }}
+    </Mutation>
+  );
+};
+const UpdateGithubUsername = props => {
+  let input;
+  return (
+    <Mutation mutation={UPDATE_GITHUB_USERNAME}>
+      {(updateGithubUsername, { data, error, loading, called }) => {
+        if (error) {
+          return <div>{JSON.stringify(error)}</div>;
+        }
+        if (loading) {
+          return <div>Submitting...</div>;
+        }
+        if (called) {
+          return <div>Github Username Updated!</div>;
+        }
+        return (
+          <div>
+            <form
+              className="w-full max-w-sm"
+              onSubmit={e => {
+                e.preventDefault();
+                updateGithubUsername({
+                  variables: { githubUsername: input.value }
+                });
+                input.value = "";
+              }}
+            >
+              <div className="flex items-center border-b border-b-2 border-teal py-2">
+                <input
+                  className="appearance-none bg-transparent border-none w-full text-grey-darker mr-3 py-1 px-2 leading-tight focus:outline-none"
+                  type="text"
+                  ref={node => {
+                    input = node;
+                  }}
+                  placeholder={props.githubUsername}
+                  aria-label="Github Username"
+                />
+                <button
+                  className="flex-no-shrink bg-teal hover:bg-teal-dark border-teal hover:border-teal-dark text-sm border-4 text-white py-1 px-2 rounded"
+                  type="submit"
+                >
+                  Update Github Username
+                </button>
+              </div>
+            </form>
+          </div>
+        );
+      }}
+    </Mutation>
+  );
+};
+const UpdateGithubOrg = props => {
+  let input;
+  return (
+    <Mutation mutation={UPDATE_GITHUB_ORG}>
+      {(updateGithubOrg, { data, error, loading, called }) => {
+        if (error) {
+          return <div>{JSON.stringify(error)}</div>;
+        }
+        if (loading) {
+          return <div>Submitting...</div>;
+        }
+        if (called) {
+          return <div>Github Org Updated!</div>;
+        }
+        return (
+          <div>
+            <form
+              className="w-full max-w-sm"
```

Comment:
Can we make this form wider?  It is cutting off the org's name:

![image](https://user-images.githubusercontent.com/7482329/49663247-4ab8a800-fa0b-11e8-897a-8120b13ffec7.png)